packages/@guardian/source-react-components-development-kitchen/src/components/toggle-switch/ToggleSwitch.stories.tsx

```diff
@@ -64,3 +65,18 @@ MediumWithLabel.args = {
 	size: 'medium',
 };
 asChromaticStory(MediumWithLabel);
+
+// *****************************************************************************
+
+const toggleCssOverrides = css`
+	width: 150px;
+	background-color: lightblue;
+`;
+
+export const limitedWidth = Template.bind({});
```

Comment:
I don't think we need this story (or at least not here, in Source). This seems like a use case for Dotcom and so it should probably live there

Reply:
Yeaa I was thinking so too, removed the story